Dockerfile

```diff
@@ -38,6 +38,15 @@ RUN apt-get update \
 WORKDIR /app
 COPY --from=0 /usr/src/java-app/*.jar ./
 
+LABEL \
+    org.label-schema.schema-version="1.0" \
+    org.label-schema.vendor="Elastic" \
+    org.label-schema.name="opbeans-java" \
+    org.label-schema.version="1.3.0" \
```

Comment:
This is not the actual version of the Java agent which is `JAVA_AGENT_BUILT_VERSION`. Is that intentional?

Reply:
The whole process to bump the dependency happens with `.ci/bump-version.sh` but it seems the agent version gets injected on the fly when building the docker image:
- https://github.com/elastic/opbeans-java/blob/37d483df81fd0b25792ebabc44693ecb67b22f84/Dockerfile#L25

My main concern with this particular agent injection on the fly is the versioning of the docker images. I understand it's backward compatible that's not my concern about but the versioning of those docker images to ensure the version of the opbeans with the agent gets a specific tag in docker for reproducibility reasons.

A bit more of context about the `.ci/bump-version.sh` script can be found in https://github.com/elastic/opbeans-java/pull/33

Our aim is standardise the release process for all the agents, even though there will be not a clear benefit in this particular case, but from the maintainability point of view it will be make things easier.

> Is that intentional?

The version 1.3.0 is not the latest one, the bump happens on a tag releases from the apm-agent-java was broken, ideally https://github.com/elastic/apm-agent-java/pull/1266 should fixed and therefore the upcoming tag releases should trigger the bump of version for this repo.

For the time being, I'll leave as it is and I'll raise another PR to do the bump manually